python/setup.cfg

```diff
@@ -28,6 +28,9 @@ build-dir  = doc/_build
 addopts = --ignore=scripts
 filterwarnings =
     error:The SparseDataFrame:FutureWarning
+    # ignore warnings when Cython functions are called
+    # from python tests
+    ignore::pytest.PytestCollectionWarning
```

Comment:
What is the warning that this would silence?

Reply:
Oh, this is a leftover from when I wanted to make Cython tests to be picked up by Pytest automatically. Didn't work, need to delete it!